src/test/java/org/broadinstitute/hellbender/tools/gvs/common/WeightedSplitIntervalsTest.java

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.hellbender.tools.gvs.common;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.OverlapDetector;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.TreeSet;
+
+
+public class WeightedSplitIntervalsTest {
```

Comment:
This should probably have at least a simple integration test to show that it actually runs and produces files with output in them?

Reply:
It would be ideal to see tests for multiple intervals / multiple chromosomes.  WeightedInterval.split() could probably use it's own tests as well.

Reply:
added a bunch of tests now!